metadata.yaml

```
@@ -0,0 +1,3 @@
+homepage: "https://matomo.org/"
+documentation: "https://github.com/matomo-org/google-tag-manager-matomo-template"
```

Should this be pointing to the event template repo?
```suggestion
documentation: "https://github.com/google-tag-manager-matomo-event-template"
```